index.js

```diff
@@ -16,6 +16,7 @@ const defaults = {
   raw: false,
   separator: ',',
   skipLines: null,
+  maxRowSize: 0,
```

Comment:
I'd like to go with `Number.MAX_SAFE_INTEGER` here instead of zero. I like to keep default options set to real-world values (doesn't always work out, but hey I can try). `0` doesn't reflect what the module does if this isn't set. Using the max value is more in line with that, and doesn't cost us anything.

Reply:
Which would fix the maximum bytes on 8 peta byte - which should be enough for the foreseeable future. I'd like to see the document having such a big row ...